File: StudentSystem.Models/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace StudentSystemDbContext.Models
{
    public class Student : ApplicationUser
    {
        public int FacultyNumber { get; set; }

        public Specialty Specialty { get; set; }

        public Faculty Faculty { get; set; }

        public DiplomaWork DiplomaWork { get; set; }
    }
}

File: db/models/Student.js

module.exports = function (sequelize, DataTypes) {
    const Sale = sequelize.define('Student', {
        id: {
            type: DataTypes.INTEGER,
            autoIncrement: true,
            primaryKey: true
        },
        username: {
            type: DataTypes.STRING
        },
        passwordHash: {
            type: DataTypes.STRING
        },
        facultyNumber: {
            type: DataTypes.INTEGER
        },
        specialty: {
            type: DataTypes.INTEGER
        },
        faculty: {
            type: DataTypes.INTEGER
        },
        diplomaWork: {
            type: DataTypes.INTEGER
        }
    });
    
    // Class Method
    Sale.associate = function (models) {
    };
    
    // Instance Method
    // Sale.prototype.someMethod = function () {
    // };

    return Sale;
};